Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -8,12 +9,58 @@ public static class BinaryConverter
     {
         public static long ToInteger(string binary)
         {
-            return 0;
+            if (string.IsNullOrEmpty(binary))
+                return 0;
+            
+            if (!binary.Contains("0") && !binary.Contains("1"))
+                throw new InvalidBinaryNumberException(binary);
+
+            double number = 0;
+            char[] binaryNumbers = binary.ToCharArray();
+
+            for (int i = binaryNumbers.Length -1; i >= 0; i--)
+            {
+                int a = (binaryNumbers.Length - (i + 1));
+                double power = Math.Pow(2, a);
+
+                number +=Char.GetNumericValue(binaryNumbers[i]) * power;
+            }
+
+            return long.Parse(number.ToString());
         }
 
         public static string ToBinary(long number)
         {
-            return "";
+            string binaryValue = "";
```

Comment:
Consider using a `StringBuilder` and returning `StringBuilder.ToString()` at the end.